api/search.js

```diff
@@ -132,21 +136,24 @@ function setup( addressDbPath, streetDbPath ){
 
       // if distance = 0 then we can simply use either A or B (they are the same lat/lon)
       // else we interpolate between the two positions
-      var point = A;
+      var point2 = A;
```

Comment:
how about "interpolatedPoint" ?

(but also, why was this renamed?)

Reply:
there is another variable named `point` which is being shadowed by this variable when using the same name.